README.md

```diff
@@ -118,26 +118,250 @@ services:
+Pebble is invoked using `pebble <command>`. To get more information:
 
-    $ pebble run
+* To see a help summary, type `pebble -h`.
+* To see a short description of all commands, type `pebble help --all`.
+* To see details for one command, type `pebble help <command>` or `pebble <command> -h`.
 
-This will start the pebble daemon itself, and start all default services as well. Then
-other pebble commands may be used to interact with the running daemon.
+A few of the commands that need more explanation are detailed below.
 
-For example, to see any recent changes, for this or previous runs, use:
+### Running the daemon (server)
 
-    $ pebble changes
+If pebble is installed and the `$PEBBLE` directory is set up, running the daemon is easy:
 
-And start or stop a specific service with:
+```
+$ pebble run
+2022-10-26T01:18:26.904Z [pebble] Started daemon.
+2022-10-26T01:18:26.921Z [pebble] POST /v1/services 15.53132ms 202
+2022-10-26T01:18:26.921Z [pebble] Started default services with change 50.
+2022-10-26T01:18:26.936Z [pebble] Service "srv1" starting: sleep 300
+```
+
+This will start the pebble daemon itself, as well as starting all the services that
+are marked as `startup: enabled` (if you don't want that, use `--hold`). Then
+other pebble commands may be used to interact with the running daemon, for example,
+in another terminal window.
+
+To override the default configuration directory, set the `PEBBLE` environment variable when running:
+
+```
+$ PEBBLE=~/pebble pebble run
+2022-10-26T01:18:26.904Z [pebble] Started daemon.
+...
+```
+
+The daemon's service manager stores the most recent stdout and stderr from each service (using a 100KB ring buffer per service). These are accessible via the logs API or using `pebble logs`. If you want to also write service logs to Pebble's own stdout, run the daemon with `--verbose`:
+
+```
+$ pebble run --verbose
+2022-10-26T01:41:32.805Z [pebble] Started daemon.
+2022-10-26T01:41:32.835Z [pebble] POST /v1/services 29.743632ms 202
+2022-10-26T01:41:32.835Z [pebble] Started default services with change 7.
+2022-10-26T01:41:32.849Z [pebble] Service "srv1" starting: python3 -u /path/to/srv1.py
+2022-10-26T01:41:32.866Z [srv1] This is log number 0
+2022-10-26T01:41:35.870Z [srv1] This is log number 1
+2022-10-26T01:41:38.873Z [srv1] This is log number 2
+...
+```
+
+### Starting and stopping services
+
+To start or stop specific services, type `pebble start` or `pebble stop` followed by one or more service names, for example:
+
+```
+$ pebble start srv1 srv2  # start two services (and any dependencies)
+
+$ pebble stop srv1        # stop one service
+```
+
+When starting a service, Pebble records a "Start" [change](#changes-and-tasks) for the operation, executes the service's `command`, and waits 1 second to ensure the command doesn't exit too quickly. Assuming the command doesn't exit within 1 second, the start is considered successful, otherwise `pebble start` will exit with an error.
```

Comment:
It seems premature to dive so deeply into the details of process management here. I'd reserve such details for an advanced section, and even then probably avoid promising things like exact timings as that can easily change.

Reply:
I'm refactored this section a little, but I disagree that this is too much detail here. This README is what we have for a user manual right now, so I think it should explain it, not try to stuff important details in an "advanced" section.

I at first removed the exact timings, but the current timings are part of said user manual, and as such should be documented. I think it's important, otherwise if we say something like "waits a short time" users will be thinking, "Okay, but how long is short?"